packages/stylelint-config/__tests__/commenting.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' ),
+	stylelint = require( 'stylelint' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( '../' ),
+	validCss = fs.readFileSync(
+		'./packages/stylelint-config/__tests__/commenting-valid.css',
```

Comment:
Aside: we mostly use `/test/` folder for tests but since `/__tests__/` is allowed as well, it's probably fine to leave it as is to avoid issues.